src/components/App/index.js

```diff
@@ -17,27 +17,18 @@ if (typeof window !== 'undefined') {
   });
 }
 
-export default class App extends Component {
-  state = {
-    isHome: true,
-  };
-
-  handleRoute = (event) => {
-    this.setState({
-      isHome: event.url === routeMap.home,
-    });
-  };
-
-  render() {
-    const { isHome } = this.state;
-    return (
-      <Wrapper isHome={isHome} id='app'>
-        {!isHome && (<Header Dark />)}
-        <Router onChange={this.handleRoute}>
+export default () => (
+  <Match path={routeMap.home}>
+    {({ url }) => (
```

Comment:
I think that rather than using the `url` it would be wiser to destructurize the `matches` property and assign it to `isHome`.
```
{({ matches: isHome}) => (
```
After that Inside components: `<Wrapper isHome={isHome}...` and `{!isHome && <Header Dark />`